polaris-react/src/components/RangeSlider/components/SingleThumb/SingleThumb.scss

```diff
@@ -220,13 +220,10 @@ $range-output-translate-x: calc(
     }
   }
   // stylelint-enable selector-max-specificity, selector-max-combinators, selector-max-class
-}
 
-.OutputText {
-  @include text-style-subheading;
-  display: block;
-  flex: 1 1 auto;
-  margin: auto;
-  text-align: center;
-  color: var(--p-text);
+  > :first-child {
+    display: block;
```

Comment:
Great point! @kyledurand and I were discussing this and he started an [issue](https://github.com/Shopify/polaris/issues/7618) to track examples like this where we need to use selectors to get elements because we aren't able to apply styles with the classnames 👍